src/utility/utils.js

```diff
@@ -1,7 +1,41 @@
+const moment = require('moment');
+const mongoose = require('mongoose');
+const Event = require('../models/event');
+const variables = require('./variables');
+
 function getFileExtension(fileName) {
     return fileName.split('.').pop();
 }
 
+async function addDummyEventData(numOfEvents) {
+    const mockEvents = [];
+    // eslint-disable-next-line no-plusplus
+    for (let i = 1; i < numOfEvents + 1; i++) {
+        mockEvents.push({
+            publisherId: mongoose.Types.ObjectId(),
+            title: `Event ${i}`,
+            content: `Event ${i} content`,
+            coverImage:
+                'https://images.unsplash.com/photo-1518791841217-8f162f1e1131?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=800&q=60',
+            date: moment('2022-01-01', 'YYYY-MM-DD')
+                .add(i - 1, 'days')
+                .format('YYYY-MM-DD'), // adding i days to the current date
+            categories: variables.CATEGORIES.slice(i - 1, i + 1),
```

Comment:
what if the i+1in slice function is bigger than the number of categories provided in the variables/categories file?